site/css/style.css

```
@@ -1135,109 +1467,1201 @@ section { padding: 5rem 0; }
+  border-left: 3px solid var(--steel-bright);
+}
+.wizard-step-tutorial[hidden] { display: none; }
+
+.wizard-step-tutorial::before {
+  content: '';
+  display: inline-block;
+  width: 14px;
+  height: 14px;
+  margin-right: 0.5rem;
+  vertical-align: -2px;
+  background: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)